tests/manual/testinput.html

```diff
@@ -45,16 +45,22 @@
         </script>
     </head>
     <body onload="init()">
-    <table>
-        <tr><td>tel:</td><td><input type="tel"></td></tr>
-        <tr><td>number:</td><td><input type="number"></td></tr>
-        <tr><td>datetime:</td><td><input type="datetime"></td></tr>
-        <tr><td>email:</td><td><input type="email"></td></tr>
-        <tr><td>password:</td><td><input type="password"></td></tr>
-        <tr><td>text:</td><td><input type="text"></td></tr>
-    </table>
-    <textarea id="area" style="width: 100%; height: 150px;"></textarea>
-    <br/>
-    <input type="button" onclick="area.value = ''" value="clear log"/>
+        <table id="inputs">
+            <tr><td>tel:</td><td><input type="tel"></td></tr>
+            <tr><td>number:</td><td><input type="number"></td></tr>
+            <tr><td>datetime:</td><td><input type="datetime"></td></tr>
+            <tr><td>email:</td><td><input type="email"></td></tr>
+            <tr><td>password:</td><td><input type="password"></td></tr>
+            <tr><td>text:</td><td><input type="text"></td></tr>
+        </table>
+        <textarea id="area" style="width: 100%; height: 150px;"></textarea><br/>
+        <input type="button" onclick="area.value = ''" value="clear log"/><br/>
+        <br/>
+        text:<br/>
+        <input type="text"><br/>
+        text, autocomplete="off":<br/>
+        <input type="text" autocomplete="off"><br/>
```

Comment:
Please add also an input field for autocomplete="none". Also "characters" would be cool even though vkb may not handle that hint.